Nodejs/Product/TestAdapter/TestAdapter.csproj

```diff
@@ -14,11 +14,11 @@
     </NuGetPackageImportStamp>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
   </PropertyGroup>
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);NOVS;NO_WINDOWS</DefineConstants>
```

Comment:
>NOVS [](start = 40, length = 4)

One has an underscore and the other doesn't?

Reply:
more refactoring coming to fix.